indigo/release/Dockerfile

```diff
@@ -5,7 +5,6 @@ FROM osrf/ros:indigo-desktop
 MAINTAINER Dave Coleman dave@dav.ee
 
 # apt-commands are combined in single RUN statement with lists folder removal to reduce image size
-RUN apt-get update \
- && apt-get upgrade -y \
```

Comment:
i decided to remove it - i like the agility:
https://github.com/davetcoleman/moveit_docker/commit/bb2068edf4a095a27befb1f33f804d5fc9045b3f